tests/svgmin.js

```diff
@@ -30,6 +30,7 @@ function makeTest(options, file, additionalSetup = () => {}) {
             // Save the transform's data to the Promise's result.
             transform.on('data', (data) => {
                 result = data.contents ? String(data.contents) : data.contents;
+                console.log(result);
```

Comment:
Can you get rid of this log?

Reply:
Oops thanks for the catch. Pushed and removed